Makefile

```diff
@@ -1,18 +1,19 @@
 
 DOCKER ?= $(shell which docker)
 DOC_FILES := \
-	README.md \
+	bundle.md \
 	code-of-conduct.md \
+	config-linux.md \
+	config.md \
+	glossary.md \
+	implementations.md \
 	principles.md \
-	style.md \
+	project.md \
```

Comment:
I should add a comment, but these files were in an order that determines how they show up in the PDF/HTML docs. 
If you could just put the project.md entry where it corresponds to where you put it in the table of contents. Thanks.